shiny/global-sensitivity/helper.R

```diff
@@ -0,0 +1,136 @@
+library(shiny)
+library(PEcAn.DB)
+library(RPostgreSQL)
+library(udunits2)
+library(lubridate)
+library(dplyr)
+
+options(scipen=12)
+
+betyConnect <- function(php.config="../../web/config.php") {
```

Comment:
Addressed in 8882edd. I merged the two helpers and migrated them to PEcAn.visualization, which both shiny apps now load.